test/e2e/gui/screens/shell.py

```diff
@@ -0,0 +1,131 @@
+import time
+
+import allure
+
+import driver
+from gui.elements.button import Button
+from gui.elements.object import QObject
+from gui.elements.text_edit import TextEdit
+from gui.objects_map import shell_names
+from gui.objects_map.names import statusDesktop_mainWindow
+from gui.screens.community_portal import CommunitiesPortal
+from gui.screens.messages import MessagesScreen
+from gui.screens.settings import SettingsScreen
+from gui.screens.wallet import WalletScreen
+
+class ShellScreen(QObject):
+
+    def __init__(self):
+        super().__init__(shell_names.shell_container)
+        self.search_field = TextEdit(shell_names.shell_search_field)
+        self.grid = QObject(shell_names.shell_grid)
+        self.dock = QObject(shell_names.shell_dock)
+
+    @allure.step('Search in Shell')
+    def search(self, query: str):
+        """Enter search query in the Shell search field"""
+        self.search_field.clear()
+        self.search_field.type_text(query)
+
+    @allure.step('Clear Shell search')
+    def clear_search(self):
+        """Clear the search field"""
+        self.search_field.clear()
+
+    @allure.step('Get search text')
+    def get_search_text(self) -> str:
+        """Get current search text"""
```

Comment:
same, comment is overkill and also we can get the text property right away, no need to wrap this into additional method